# HG changeset patch
# User Torsten <mttl@mailbox.org>
# Date 1489826636 -3600
#      Sat Mar 18 09:43:56 2017 +0100
# Node ID 5bddd4089386b127452a2368a3e4dda191efac29
# Parent  c9245d94aac56edd8d2d1ddb3ac787146546824a
* file-editor.cc: fix compiling wihtout qscintilla (bug #50572)

diff --git a/libgui/src/m-editor/file-editor.cc b/libgui/src/m-editor/file-editor.cc
--- a/libgui/src/m-editor/file-editor.cc
+++ b/libgui/src/m-editor/file-editor.cc
@@ -2463,19 +2463,16 @@ file_editor::switch_tab (int direction, 
       _tab_widget->setCurrentIndex (new_pos);
       focus ();
 #endif
     }
   else
     _tab_widget->setCurrentIndex (new_pos);
 }
 
-#endif
-
-
 //
 // Functions of the the reimplemented tab widget
 //
 
 // Reimplement mouse event for filtering out the desired mouse clicks
 void
 file_editor_tab_widget::mousePressEvent(QMouseEvent *me)
 {
@@ -2504,8 +2501,9 @@ file_editor_tab_widget::mousePressEvent(
     }
   else
     {
       // regular handling of the mouse event
       QTabWidget::mousePressEvent (me);
     }
 }
 
+#endif
